### PR TITLE
Remove live installer special handling (SR#420003)

### PR DESCRIPTION
**Pull request by okurz**
Verification run: http://lord.arch/tests/3251

### CODE REVIEW

File: tests/installation/livecd_network_settings.pm [hunk 1]
```diff
@@ -17,8 +17,8 @@ use testapi;
 # 'network settings' after welcome
 sub run() {
     assert_screen 'inst-network_settings-livecd';
-    # ne'x't
-    send_key 'alt-x';
+    # 'n'ext
+    send_key 'alt-n';
```
Comment (DimStar77) [inline tests/installation/livecd_network_settings.pm]: wouldn't that just be send_key $cmd{next}; ?

Comment (okurz) [inline tests/installation/livecd_network_settings.pm]: yes, it looks like there are many more occassions of alt-n which should be $cmd{next}. I will try to change all of these and provide it in an explicit commit.

Comment (okurz) [inline tests/installation/livecd_network_settings.pm]: updated

**Comment by DimStar77** [comment]
LGTM

**Pull request merged**